packages/terra-site/src/examples/form-input/DefaultErrorExample.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import FormInput from 'terra-form-input/lib/FormInput';
+
+class DefaultErrorExample extends React.Component {
```

Comment:
Is the point of the example to show what an inValid input looks like? If so, I'd just make it an uncontrolled input.

Reply:
Addressed in 3457238